ext/js/display/display-anki.js

```diff
@@ -1303,6 +1326,34 @@ export class DisplayAnki {
 
         return 'linear-gradient(to right,' + gradientSlices.join(',') + ')';
     }
+
+    /**
+     * Get the highest priority state from a list of Anki queue states.
+     *
+     * Priority order:
+     *   - -3, -2 → "buried"
+     *   - -1 → "suspended"
+     *   -  2 → "review"
+     *   -  1, 3 → "learning"
+     *   -  0 → "new" (default fallback)
+     * @param {number[]} cardStates Array of queue state integers.
+     * @returns {"buried" | "suspended" | "review" | "learning" | "new"} - The highest priority state found.
+     */
+    _getHighestPriorityCardState(cardStates) {
+        if (cardStates.includes(-3) || cardStates.includes(-2)) {
+            return 'buried';
+        }
+        if (cardStates.includes(-1)) {
+            return 'suspended';
+        }
+        if (cardStates.includes(2)) {
+            return 'review';
+        }
+        if (cardStates.includes(1) || cardStates.includes(3)) {
+            return 'learning';
+        }
+        return 'new';
```

Comment:
This needs to not default to `new`. If `noteInfos` is not searched for, Yomitan will always say `new` on the view notes button. Maybe make this null or empty string and add a case where the state isnt shown in the tooltip. Maybe add back `view-note.svg` so it can be a clear default.